acceptance/openstack/objectstorage/v1/objects_test.go

```diff
@@ -82,6 +84,25 @@ func TestObjects(t *testing.T) {
 	}
 	th.AssertEquals(t, len(ois), len(oNames))
 
+	// Create temporary URL, download its contents and compare with what was originally created.
+	// Downloading the URL validates it (this cannot be done in unit tests).
+	objURLs := make([]string, numObjects)
+	for i := 0; i < numObjects; i++ {
+		objURLs[i], err = objects.CreateTempURL(client, cName, oNames[i], objects.CreateTempURLOpts{
+			Method: http.MethodGet,
+			TTL:    180,
+		})
+		th.AssertNoErr(t, err)
+
+		resp, err := http.Get(objURLs[i])
+		th.AssertNoErr(t, err)
+		defer resp.Body.Close()
```

Comment:
not critical, but better to call `resp.Body.Close()` at the end of the loop. otherwise go will close the connection only when the test function exits.

Reply:
Fixed in 22628a7ff541e2ff3bfab63a75f1d9ef6fd71fb6